### ISSUE TITLE
Multiple Sentient Disease Spawns Break Infecting (and a lot else)

### ISSUE THREAD

**Issue by AyamePup** [tgstation/tgstation#75314]
## Reproduction: 
Should work if you just spawn two sentient diseases at the same time. When two player controlled sentient disease are active, both are unable to infect new hosts, (other than monkeys, it seems?? which do not count towards adaptation points.)
https://github.com/tgstation/tgstation/issues/75017 seems to report a similar issue with the menu not opening- a bug I also encountered the last time two sentient diseases spawned and I was one, (unfortunately dont remember the round id- could be the one linked in the mentioned report.) This time, however, I was still able to open the menu, but unable to infect.

Basically, if more than one spawns, you're immediately fucked. Considering how sentient disease is usually ALREADY fucked 90% of the time, this kind of breaks it completely.

**Comment by Jackal-boop** [comment]
is SD worth keeping?

**Comment by Sealed101** [comment]
it do be a dupe of #64329
